.travis.yml

```diff
@@ -148,11 +148,13 @@ after_success:
         echo "HOST_CLOUD is NOT set to one of the following values:aws, azure, gce, openstack."
     fi
 
-  # The below script used to be run as post-processor for the Openstack's builder.
+  # The below script for Openstack used to be run as post-processor for the Openstack's builder.
   # In order to avoid running Terraform within Packer, hence getting a cumbersome and tedious log output.
   - >
-    if [ $HOST_CLOUD = 'openstack' ]; then
+    if [ "$HOST_CLOUD" = 'openstack' ]; then
         travis_retry bin/os_pp.sh
+    elif [ "$HOST_CLOUD" = 'aws' ]; then
+        bin/aws_pp.sh
```

Comment:
Script that will copy the latest created AMI into the other regions. I've decided to put it here instead of a typical packer post-processor for two main reasons:

1. Reduce verbosity
2. Simple solution to extract the created AMI ID (to be used in the `aws_pp.sh` script)